src/align.py

```diff
@@ -16,8 +16,28 @@ def align(x: str, y: str, edits: str) -> tuple[str, str]:
     ('ACCACAGT-CATA', 'A-CAGAGTACAAA')
 
     """
-    return "", ""
-
+    x, y = iter(x), iter(y)
+    seq1, seq2 = list(), list()
+    for i, edit in enumerate(edits):
```

Comment:
You are not using `i`, so `for edit in edits:` would do, I think.

Reply:
Okay! Thank you for the feedback